application-flashmessages-ui/src/main/resources/Flash/DisplayFlashMessages.xml

```diff
@@ -275,7 +275,7 @@
 #flashMessages {
   background-color: $theme.menuBackgroundColor;
   box-shadow: 0 2px 8px rgba(0, 0, 0, 0.4) inset;
-  color: #fff;
+  color: $theme.menuLinkColor;
```

Comment:
ok thanks so we can't add a dependendy on this since xwiki-platform-flamingo-skin-resources (which contains colorThemeInit.vm) is a pom packaging which produces a zip which is then unzipped later in the build. So it's fine. thanks.